test/test.go

```diff
@@ -126,12 +129,14 @@ func ParseTestResults(testsRootDir string) (results Results, err error) {
 
 	// find test results in each dir, skip if invalid test dir
 	for _, testDir := range testDirs {
+		// <root_tests_dir>/<test_dir>
+		testDirPath := filepath.Join(testsRootDir, testDir.Name())
+
 		if !testDir.IsDir() {
+			log.Debugf(colorstring.Yellowf("%s is not a directory", testDirPath))
```

Comment:
the step has no input for turning on debug log level, so either:
- wire in input for it
- use another log severity
- remove the debug logs